### PR TITLE
Update builder examples for publisher and subscriber in the README

### PR DESCRIPTION
**Pull request by KacperKluka**
I've updated the examples of creating publisher and subscriber in our README.

### CODE REVIEW

**Review by tbedford — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -196,11 +217,19 @@ Here is an example of how Asset Subscribing SDK can be used:
...
+    .trackingId(trackingId) // provide the tracking identifier for the asset that needs to be tracked
+    // Optional configuration
     .resolution( // request a specific resolution to be considered by the publisher
-        Resolution(Accuracy.MAXIMUM, desiredInterval = 1000L, minimumDisplacement = 1.0)
+      Resolution(Accuracy.MAXIMUM, desiredInterval = 1000L, minimumDisplacement = 1.0)
     )
-    .trackingId(trackingId) // provide the tracking identifier for the asset that needs to be tracked
+    .logHandler(object : LogHandler {
+      override fun logMessage(level: LogLevel, message: String, throwable: Throwable?) {
+        // log the message to internal or external loggers
```
Comment (QuintinWillison) [inline README.md:229]:
```suggestion
        // TODO: log the message to internal or external loggers
```

File: README.md [hunk 2]
```diff
@@ -166,11 +166,32 @@ val defaultResolution = Resolution(Accuracy.BALANCED, desiredInterval = 1000L, m
...
+            // create the notification for location updates background service
+        }
+      },
+      NOTIFICATION_ID
+    )
+    // Optional configuration
     .profile(RoutingProfile.DRIVING) // provide mode of transportation for better location enhancements
+    .logHandler(object : LogHandler {
+        override fun logMessage(level: LogLevel, message: String, throwable: Throwable?) {
+          // log the message to internal or external loggers
```
Comment (QuintinWillison) [inline README.md:186]:
```suggestion
          // TODO: log the message to internal or external loggers
```

File: README.md [hunk 3]
```diff
@@ -166,11 +166,32 @@ val defaultResolution = Resolution(Accuracy.BALANCED, desiredInterval = 1000L, m
...
 val publisher = Publisher.publishers() // get the Publisher builder in default state
+    // Required configuration
     .connection(ConnectionConfiguration(Authentication.basic(CLIENT_ID, ABLY_API_KEY))) // provide Ably configuration with credentials
     .map(MapConfiguration(MAPBOX_ACCESS_TOKEN)) // provide Mapbox configuration with credentials
     .androidContext(this) // provide Android runtime context
     .resolutionPolicy(DefaultResolutionPolicyFactory(defaultResolution, this)) // provide either the default resolution policy factory or your custom implementation
+    .backgroundTrackingNotificationProvider(
+      object : PublisherNotificationProvider {
+        override fun getNotification(): Notification {
+            // create the notification for location updates background service
```
Comment (QuintinWillison) [inline README.md:177]:
```suggestion
            // TODO: create the notification for location updates background service
```

**Review by QuintinWillison — CHANGES_REQUESTED** [review]
A couple of small suggests to make it really clear for the app developer where they still need to inject their own code.

**Review by QuintinWillison — APPROVED** [review]

**Pull request merged**